corehq/apps/userreports/README.rst

```diff
@@ -486,6 +486,12 @@ The following operators are currently supported:
 | ``gte``         | is greater than   | number          | ``doc["age |
 |                 | or equal          |                 | "] >= 21`` |
 +-----------------+-------------------+-----------------+------------+
+| ``regex``       | matches regular   | string          | ``re.searc |
+|                 | expression        |                 | h("^([Rr]e |
+|                 |                   |                 | d|[Bb]lue) |
+|                 |                   |                 | $", doc["c |
+|                 |                   |                 | olor"])``  |
++-----------------+-------------------+-----------------+------------+
```

Comment:
Here's the reformatted if you want to update (I used https://pandoc.org/try/):
```
+------------------+---------------------------------------------+------------+-------------------------------------------------------+
| Operator         | Description                                 | Value type | Example                                               |
+==================+=============================================+============+=======================================================+
| ``eq``           | is equal                                    | constant   | ``doc["age"] == 21``                                  |
+------------------+---------------------------------------------+------------+-------------------------------------------------------+
| ``not_eq``       | is not equal                                | constant   | ``doc["age"] != 21``                                  |
+------------------+---------------------------------------------+------------+-------------------------------------------------------+
| ``in``           | single value is in a list                   | list       | ``doc["color"] in ["red", "blue"]``                   |
+------------------+---------------------------------------------+------------+-------------------------------------------------------+
| ``in_multi``     | a value is in a multi select                | list       | ``selected(doc["color"], "red")``                     |
+------------------+---------------------------------------------+------------+-------------------------------------------------------+
| ``any_in_multi`` | one of a list of values in in a multiselect | list       | ``selected(doc["color"], ["red", "blue"])``           |
+------------------+---------------------------------------------+------------+-------------------------------------------------------+
| ``lt``           | is less than                                | number     | ``doc["age"] < 21``                                   |
+------------------+---------------------------------------------+------------+-------------------------------------------------------+
| ``lte``          | is less than or equal                       | number     | ``doc["age"] <= 21``                                  |
+------------------+---------------------------------------------+------------+-------------------------------------------------------+
| ``gt``           | is greater than                             | number     | ``doc["age"] > 21``                                   |
+------------------+---------------------------------------------+------------+-------------------------------------------------------+
| ``gte``          | is greater than or equal                    | number     | ``doc["age"] >= 21``                                  |
+------------------+---------------------------------------------+------------+-------------------------------------------------------+
| ``regex``        | matches regular expression                  | string     | ``re.search("^([Rr]ed|[Bb]lue)$", doc["color"])``     |
+------------------+---------------------------------------------+------------+-------------------------------------------------------+
```

Reply:
I wasn't sure how strict rst formatting is - this looks much nicer.